flow/util/genMassive.py

```diff
@@ -436,16 +436,18 @@ def writeConfigs(CurAttrs, CurChunkNum):
 MakeArg = sys.argv[1]
 
 
-if not os.path.isdir('./metrics_%s'%ShellName):
-  os.mkdir('./metrics_%s'%ShellName)
+if not os.path.isdir('./metrics'):
+  os.mkdir('./metrics')
+if not os.path.isdir('./metrics/metrics_%s'%ShellName):
+  os.mkdir('./metrics/metrics_%s'%ShellName)
 
 knobs = assignEmptyAttrs(SweepingAttributes)
 ProductAttrs = list(productDict(knobs))
 writeDoeLog(SweepingAttributes, ProductAttrs)
 if os.path.isfile('./%s.sh'%ShellName):
   os.remove('./%s.sh'%ShellName)
-if os.path.isfile('./%s_metrics_collect.sh'%ShellName):
-  os.remove('./%s_metrics_collect.sh'%ShellName)
+if os.path.isfile('./metrics/%s_metrics_collect.sh'%ShellName):
+  os.remove('./metrics/%s_metrics_collect.sh'%ShellName)
```

Comment:
set a variable once use everywhere for `%s_metrics_collect.sh` and review for other opportunities